### DISCUSSION TITLE
Using what's on hand. No microphone. Just a webcam.

### DISCUSSION THREAD

**Discussion by mpfl** [mcguirepr89/BirdNET-Pi#452] [Show and tell]
I love to dive right into projects and kludge things I have on hand together to get stuff running. It can take a while for the right parts to get to me in this isolated city.

I discovered BirdNET-Pi this morning. I have some Pi4s left over from when I thought it would be funny to build a Kubernetes cluster. I also have a couple of Logitech C270 webcams that I was planning to use to monitor 3D printers.

The C270 has been surprisingly effective at picking up some of the birds that live around me... I definitely want a better microphone as I can hear birds that BirdNET-Pi is not detecting. But a cheap webcam can work as a proof of concept.

![Screenshot 2022-07-16 164607](https://user-images.githubusercontent.com/7235428/179347760-1e6e1652-7c01-40c8-9ed6-8315e7135bbc.png)

The spotted dove loves sitting on the TV aerial on the house next to mine. It (he?) will call nonstop for days. I guess it's that time of year...

**Comment by nilspupils** [comment]
That's the beauty of it, it already works with a simple setup out of the box. But you will be surprised how much it can pick up with a better mic. Have fun with BirdNET-Pi!
